cmd/gazelle/diff_test.go

```diff
@@ -102,6 +102,18 @@ func TestDiffNew(t *testing.T) {
 	testtools.CheckFiles(t, dir, want)
 }
 
+func TestDiffEmpty(t *testing.T) {
+	files := []testtools.FileSpec{
+		{Path: "WORKSPACE"},
+	}
+	dir, cleanup := testtools.CreateFiles(t, files)
+	defer cleanup()
+
+	if err := runGazelle(dir, []string{"-go_prefix=example.com/hello", "-mode=diff", "-patch=p"}); err != nil {
+		t.Fatalf("Expected no error, but got %q", err)
+	}
```

Comment:
The test should check the content of the patch file `p`. You can use `testtools.CheckFiles` like in the other tests.

Reply:
Yeah, np, I actually still need help with this test, as it doesn't fail when the change is reverted. That's why I haven't updated it yet.

I can't figure out a way to verify that it'd fail without the proposed change. Any thoughts are appreciated.

Reply:
nevermind, figured it out.